.github/workflows/main.yml

```diff
@@ -27,6 +27,7 @@ jobs:
       COMPOSER_VERSION: 2
       INSTALL_PATH: php-cypher-dsl
       REPOSITORY_NAME: php-cypher-dsl
+      FIXER_RULES: @PSR12,align_multiline_comment,array_indentation,blank_line_before_statement,indentation_type,no_trailing_whitespace,no_unused_imports,no_useless_else,no_useless_return,ordered_imports,phpdoc_scalar,return_assignment,simplified_if_return,single_line_after_imports,switch_case_space,trailing_comma_in_multiline,ternary_operator_spaces
```

Comment:
Use quotes here to fix the YAML syntax error.

Reply:
tnx!